### DISCUSSION TITLE
Console2Desk

### DISCUSSION THREAD

**Discussion by Panz3rFaust78** [Special-Niewbie/HandleOS#3] [General]
hi and great job!! i have installed handleOS on my ayaneo kun and works great...the ayaspace app work proprerly....I wanted to ask if there was the possibility when switching between desktop and console (e viceversa)  to automatically close the 2desk screen (remains open in the center of the screen)...it's absolutely not a problem..more of a convenience.

**Comment by Special-Niewbie** [comment]
Hi @Panz3rFaust78 really happy to hear that  :) ,and welcome on board, soon I will announce the Discord server as well, in menatime I send you the invitation https://discord.gg/2Kcqxwrs .

About C2D, would be better to keep open in the process because on swithing between Desktop Mode e Console Mode,  it's making some refresh in background and control all the processes related to dont' encountering the problems. Specially the latest Update 2.7.0, I disabled to intercat with C2D in menatime it's making the `Switching` movements to don't let mistakenly press some buttons from the final user. ;).
